gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -67,34 +73,41 @@ def _get_platform_pca_ht_path(part: str, version: str = CURRENT_VERSION) -> str:
 def _get_ancestry_pca_ht_path(
     part: str,
     version: str = CURRENT_VERSION,
-    include_unreleasable_samples: bool = False,
+    remove_unreleasable_samples: bool = True,
```

Comment:
my original comment about  "--include-unreleasable-samples"  was just referring to the argparse part, but also fine with changing the resources